.eslintrc.js

```diff
@@ -1,3 +1,22 @@
 module.exports = {
-    "extends": "google"
+  'env': {
+    'es6': true,
```

Comment:
missing environments. If you dont fix it will fail on our server side files and also on our test files
[check documentation](https://eslint.org/docs/user-guide/configuring#specifying-environments)